Library/Formula/highlighting-kate.rb

```diff
@@ -0,0 +1,35 @@
+require "language/haskell"
+
+class HighlightingKate < Formula
+  include Language::Haskell::Cabal
+
+  homepage "https://github.com/jgm/highlighting-kate"
+  url "https://github.com/jgm/highlighting-kate/archive/0.5.14.tar.gz"
+  sha256 "0c9ad39ee1b123b655434ff3a189c7aae634057711e120c08c01e4c310e959fd"
+
+  head "https://github.com/jgm/highlighting-kate.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "gmp"
+
+  fails_with :clang do
+    build 425
+    cause "clang segfaults on Lion"
+  end
+
+  def install
+    cabal_sandbox do
+      cabal_install "--only-dependencies"
+      system "make", "prep"
+      cabal_install "--prefix=#{prefix}", "-fsplitBase", "-fexecutable"
+    end
+    cabal_clean_lib
+
+    File.rename(bin/"Highlight", bin/"highlighting-kate") # Conflicts.
```

Comment:
@mikemcquaid It turns out it won't be necessary at all — got it changed upstream.